src/Repository/HealthCheckConfigRepository.php

```diff
@@ -0,0 +1,156 @@
+        Connection $connection,
+        $dbPrefix
+    ) {
+        $this->connection = $connection;
+        $this->dbPrefix = $dbPrefix;
+    }
+
+    /**
+     * @return int|null
+     *
+     * @throws DatabaseException
+     * @throws \Doctrine\DBAL\DBALException
+     */
+    public function getLastId()
+    {
+        $query = "SELECT MAX(id_health_check_config) as id FROM `{$this->dbPrefix}health_check_config`";
+
+        $statement = $this->connection->executeQuery($query);
+        if (0 === (int) $statement->errorCode()) {
+            return $statement->fetchColumn(0);
+        }
+    }
+
+    /**
+     * @throws DatabaseException
+     */
+    public function update(array $data)
+    {
+        $qb = $this->connection->createQueryBuilder();
+        $qb
+            ->update($this->dbPrefix . 'health_check_config')
+            ->set('token', ':token')
+            ->set('ips', ':ips')
+            ->setParameters([
+                'token' => $data['token'],
+                'ips' => $data['ips'],
+            ])
+        ;
+        $this->executeQueryBuilder($qb, 'HealthCheck config error');
+    }
+
+    /**
+     * @return array
+     *
+     * @throws \Doctrine\DBAL\DBALException
+     */
+    public function createTables()
+    {
+        $engine = _MYSQL_ENGINE_;
+        $this->dropTables();
+
+        $query = "CREATE TABLE IF NOT EXISTS `{$this->dbPrefix}health_check_config`(
+    			`id_health_check_config` int(10) unsigned NOT NULL auto_increment,
+    			`token` varchar(255) default NULL,
+    			`ips` varchar(255) default NULL,
+    			PRIMARY KEY (`id_health_check_config`)
+            ) ENGINE=$engine DEFAULT CHARSET=utf8";
+
+        $statement = $this->connection->executeQuery($query);
+        if (0 != (int) $statement->errorCode()) {
+            return [
+                'key' => json_encode($statement->errorInfo()),
+                'parameters' => [],
+                'domain' => 'Admin.Modules.Notification',
+            ];
+        }
+
+        return [];
+    }
+
+    /**
+     * @return array
+     *
+     * @throws \Doctrine\DBAL\DBALException
+     */
+    public function installFixtures()
+    {
+        $query = sprintf('INSERT INTO `%shealth_check_config` (`id_health_check_config`, `token`, `ips`) VALUES
+                (1, \'%s\', \'127.0.0.1, ::1\');', $this->dbPrefix, \Tools::passwdGen(128, 'RANDOM'));
+
+        $statement = $this->connection->executeQuery($query);
+        if (0 != (int) $statement->errorCode()) {
+            return [
+                'key' => json_encode($statement->errorInfo()),
+                'parameters' => [],
+                'domain' => 'Admin.Modules.Notification',
+            ];
+        }
+
+        return [];
+    }
+
+    /**
+     * @return array
+     *
+     * @throws \Doctrine\DBAL\DBALException
+     */
+    public function dropTables()
+    {
+        $sql = 'DROP TABLE IF EXISTS ' . $this->dbPrefix . 'health_check_config';
```

Comment:
Don't mix between concatenation (with dot or ""), sprintf, etc be consistent

Reply:
:+1: